WeakAuras/GenericTrigger.lua

```diff
@@ -1624,6 +1626,10 @@ do
         lastSwingOff, swingDurationOff = nil, nil
         WeakAuras.ScanEvents("SWING_TIMER_END")
       end
+    elseif event == "PLAYER_ENTER_COMBAT" then
+      isAttacking = true
```

Comment:
That's obviously not correct. A player can enter combat without auto attacking.

Reply:
See https://wowpedia.fandom.com/wiki/PLAYER_ENTER_COMBAT

Reply:
Ah indeed.